apertium-spa.spa.dix

```diff
@@ -80700,6 +80700,390 @@
+<e lm="tronchante" a="albertonl">   <i>tronchante</i><par n="abundante__adj"/></e>
+<e lm="trotacalles" a="albertonl">   <i>trotacalles</i><par n="campus__n"/></e>
+<e lm="zahorí" a="albertonl">   <i>zahorí</i><par n="club__n"/></e>
+<e lm="zagal" a="albertonl">   <i>zagal</i><par n="señor__n"/></e>
+<e lm="vitrocerámico" a="albertonl">   <i>vitrocerámic</i><par n="absolut/o__adj"/></e>
+<e lm="vacilón" a="albertonl">   <i>vacil</i><par n="anfitri/ón__adj"/></e>
+<e lm="turronero" a="albertonl">   <i>turroner</i><par n="abogad/o__n"/></e>
+<e lm="tropel" a="albertonl">   <i>tropel</i><par n="abril__n"/></e>
+<e lm="ruborizar" a="albertonl">   <i>rubori</i><par n="abalan/zar__vblex"/></e>
+<e lm="soliviantar" a="albertonl">   <i>soliviant</i><par n="abandon/ar__vblex"/></e>
+<e lm="sonrojar" a="albertonl">   <i>sonroj</i><par n="abandon/ar__vblex"/></e>
+<e lm="taconear" a="albertonl">   <i>sonroj</i><par n="abandon/ar__vblex"/></e>
+<e lm="terciar" a="albertonl">   <i>terci</i><par n="abandon/ar__vblex"/></e>
+<e lm="tintinear" a="albertonl">   <i>sonroj</i><par n="abandon/ar__vblex"/></e>
+<e lm="trampear" a="albertonl">   <i>trampe</i><par n="abandon/ar__vblex"/></e>
+<e lm="trapichear" a="albertonl">   <i>trapiche</i><par n="abandon/ar__vblex"/></e>
+<e lm="ulular" a="albertonl">   <i>ulul</i><par n="abandon/ar__vblex"/></e>
+<e lm="universalizar" a="albertonl">   <i>universali</i><par n="abalan/zar__vblex"/></e>
+<e lm="victimizar" a="albertonl">   <i>victimi</i><par n="abalan/zar__vblex"/></e>
+<e lm="vocear" a="albertonl">   <i>voce</i><par n="abandon/ar__vblex"/></e>
+<e lm="vulgarizar" a="albertonl">   <i>vulgari</i><par n="abalan/zar__vblex"/></e>
+<e lm="yodar" a="albertonl">   <i>yod</i><par n="abandon/ar__vblex"/></e>
+<e lm="zafar" a="albertonl">   <i>zaf</i><par n="abandon/ar__vblex"/></e>
+<e lm="zapear" a="albertonl">   <i>zape</i><par n="abandon/ar__vblex"/></e>
+<e lm="zigzaguear" a="albertonl">   <i>zigzague</i><par n="abandon/ar__vblex"/></e>
+<e lm="zumbar" a="albertonl">   <i>zumb</i><par n="abandon/ar__vblex"/></e>
+<e lm="idiotizar" a="albertonl">   <i>idioti</i><par n="abalan/zar__vblex"/></e>
+<e lm="lastimar" a="albertonl">   <i>lastim</i><par n="abandon/ar__vblex"/></e>
+<e lm="liofilizar" a="albertonl">   <i>liofili</i><par n="abalan/zar__vblex"/></e>
+<e lm="lloviznar" a="albertonl">   <i>llovizn</i><par n="abandon/ar__vblex"/></e>
+<e lm="machetear" a="albertonl">   <i>machete</i><par n="abandon/ar__vblex"/></e>
+<e lm="olfatear" a="albertonl">   <i>olfate</i><par n="abandon/ar__vblex"/></e>
+<e lm="pestañear" a="albertonl">   <i>pestañe</i><par n="abandon/ar__vblex"/></e>
+<e lm="rechinar" a="albertonl">   <i>rechin</i><par n="abandon/ar__vblex"/></e>
+<e lm="retozar" a="albertonl">   <i>reto</i><par n="abalan/zar__vblex"/></e>
+<e lm="retranquear" a="albertonl">   <i>retranque</i><par n="abandon/ar__vblex"/></e>
+<e lm="ungir" a="albertonl">   <i>ung</i><par n="diri/gir__vblex"/></e>
```

Comment:
Spanish verbs are written in two lines. Look:
`<e lm="abandonar">       <i>abandon</i><par n="abandon/ar__vblex"/></e>`
`<e lm="abandonar">       <p><l>abandón</l>   <r>abandon</r></p><par n="abandón/ar__vblex"/></e>`